test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectConfigurationTests.cs

```diff
@@ -221,5 +220,18 @@ public void RoundTripFromJson()
 
             TestUtilities.AssertFailIfErrors(context);
         }
+
+        [Fact]
+        public void ConditionalCollectionSerialization()
+        {
+            var context = new CompareContext {Title = "ConditionalCollectionSerialization"};
+            // Initialize an OpenIdConnectConfiguration object with all collections empty.
+            var oidcWithEmptyCollections = new
+                OpenIdConnectConfiguration();
+            var oidcWithEmptyCollectionsJson = OpenIdConnectConfiguration.Write(oidcWithEmptyCollections);
```

Comment:
Please add a test where each collection has a value.

Reply:
Just added this in!